app/containers/zazu.js

```diff
@@ -1,66 +1,59 @@
 const React = require('react')
+const PropTypes = require('prop-types')
```

Comment:
Ah, just forgot to add the module. Strange it didn't give me an error about it... Using PropTypes from React is afaik deprecated.